hostpolicy/tests.py

```diff
@@ -13,6 +13,16 @@ def clean_and_save(entity):
     entity.save()
 
 
+class Internals(TestCase):
+    """Test internal data structures."""
+
+    def test_str(self):
+        """Test that __str__ returns obj.name."""
+        name = "test1"
+        atom = HostPolicyAtom(name=name, description='test')
+        self.assertEqual(str(atom), '"' + name + '"')
```

Comment:
This looks better with a f-string: `f'"{name}"'`

Reply:
Agreed.

Reply:
Fixed in https://github.com/unioslo/mreg/pull/484/commits/3fb4213c07bbd979d25a4cf18a83dae61abdb22b